package.json

```diff
@@ -88,12 +88,9 @@
     "migrate:up": "grunt db:migrate:up || true",
     "migrate:down": "grunt db:migrate:down",
     "uswds": "npm run uswds:css && npm run uswds:images && npm run uswds:fonts",
-    "uswds:css": "npm run uswds:css:sass && npm run uswds:css:uswds && npm run uswds:css:gfonts",
-    "uswds:css:sass": "cd node_modules/uswds && npm run build-sass && cd ../..",
-    "uswds:css:uswds": "cp node_modules/uswds/assets/css/uswds.css assets/styles/",
-    "uswds:css:gfonts": "cp node_modules/uswds/assets/css/google-fonts.css assets/styles/",
-    "uswds:images": "cp -r node_modules/uswds/assets/img/ ./assets/images/uswds",
-    "uswds:fonts": "cp node_modules/uswds/assets/fonts/* ./assets/fonts",
+    "uswds:css": "cp node_modules/uswds/dist/css/uswds.css assets/styles/",
+    "uswds:images": "cp -r node_modules/uswds/dist/img/ ./assets/images/uswds",
+    "uswds:fonts": "cp node_modules/uswds/dist/fonts/* ./assets/fonts",
```

Comment:
Here's an example https://github.com/18F/fec-eregs/blob/53a5983/Gruntfile.js#L11

Maybe copy is the simplest and the most straight forward.